README.md

```diff
@@ -1,19 +1,35 @@
 # Adobe Experience Platform - UserProfile extension for Android
 
-[![CircleCI](https://dl.circleci.com/status-badge/img/gh/adobe/aepsdk-userprofile-android/tree/dev-v2%2E0%2E0.svg?style=svg)](https://dl.circleci.com/status-badge/redirect/gh/adobe/aepsdk-userprofile-android/tree/dev-v2%2E0%2E0)
-[![codecov](https://codecov.io/gh/adobe/aepsdk-userprofile-android/branch/main/graph/badge.svg?token=UgtU9vDA6j)](https://codecov.io/gh/adobe/aepsdk-userprofile-android)
+[![Maven Central](https://img.shields.io/maven-central/v/com.adobe.marketing.mobile/userprofile.svg?logo=android&logoColor=white&label=userprofile)](https://mvnrepository.com/artifact/com.adobe.marketing.mobile/userprofile)
 
-## BETA ACKNOWLEDGEMENT
+## About this project
 
-AEPUserProfile is currently in Beta. Use of this code is by invitation only and not otherwise supported by Adobe. Please contact your Adobe Customer Success Manager to learn more.
+The [`UserProfile`](https://developer.adobe.com/client-sdks/documentation/profile/) extension represents the Adobe Experience Platform SDK's Profile extension which helps manage profile attributes in the client.
 
-By using the Beta, you hereby acknowledge that the Beta is provided "as is" without warranty of any kind. Adobe shall have no obligation to maintain, correct, update, change, modify or otherwise support the Beta. You are advised to use caution and not to rely in any way on the correct functioning or performance of such Beta and/or accompanying materials.
+## Installing the AEP UserProfile SDK for Android
 
-## About this project
+The AEP SDK supports Android API 19 (Kitkat) and newer.
+
+Installation via [Maven](https://maven.apache.org/) & [Gradle](https://gradle.org/) is the easiest and recommended way to get the AEP SDK into your Android app.  In your `build.gradle` file, include the latest version of following dependencies:
+
+```gradle
+implementation 'com.adobe.marketing.mobile:core:2.x.x'
+implementation 'com.adobe.marketing.mobile:userprofile:2.x.x'
+```
+
+## Development
 
-The Adobe Experience Platform UserProfile Mobile Extension is an extension for the [Adobe Experience Platform SDK](https://github.com/Adobe-Marketing-Cloud/acp-sdks).
+**Open the project**
 
-To learn more about this extension, read [Adobe Experience Platform Profile Mobile Extension](https://aep-sdks.gitbook.io/docs/foundation-extensions/profile).
+To open and run the project, open the `code/build.gradle` file in Android Studio
+
+**Run demo application**
```

Comment:
Saying `demo` application is a little bit confusing here, can we call it the `test` app?